src/Javis.jl

```diff
@@ -128,10 +128,17 @@ end
 """
     Translation(p::Union{Point, Symbol})
 
-Create a `Translation(p, p)` such that 
-if there is no animation the start position and end position are the same.
+Create a `Translation(O, p)` such that a translation is done from the origin.
 """
-Translation(p::Union{Point, Symbol}) = Translation(p, p)
+Translation(p::Union{Point, Symbol}) = Translation(O, p)
+
+"""
+    Translation(x::Real, y::Real)
+
+Create a `Translation(O, Point(x,y))` such that a translation is done from the origin.
+Shorthand for writing `Translation(Point(x,y))`.
+"""
+Translation(x::Real, y::Real) = Translation(Point(x, y))
 
 """
```

Comment:
Since we decided to only thoroughly document exported functions, could you add a line of an example invocation of this function? To show how it can be used ?